src/main/java/nl/tudelft/ti2206/group9/audio/SoundEffectPlayer.java

```diff
@@ -37,6 +42,7 @@ public SoundEffectPlayer(final String soundPath) {
         if (State.isSoundEffectsEnabled()) {
             initializeAudio(path);
         }
+        this.setVolume(DEFAULT_VOLUMELEVEL);
```

Comment:
This is a nice idea. Storing the default in State makes actually more sense, indeed. :)
I agree, will change. Thanks!

<review,logic